src/Nethermind/Nethermind.Runner/Ethereum/Steps/StartRpc.cs

```diff
@@ -30,8 +33,31 @@ public StartRpc(INethermindApi api)
         public async Task Execute(CancellationToken cancellationToken)
         {
             IJsonRpcConfig jsonRpcConfig = _api.Config<IJsonRpcConfig>();
+            IKeyStoreConfig keyStoreConfig = _api.Config<IKeyStoreConfig>();
             ILogger logger = _api.LogManager.GetClassLogger();
 
+
+            if (string.IsNullOrEmpty(jsonRpcConfig.JwtSecretFile) || jsonRpcConfig.JwtSecretFile == "null")
+            {
+                string defaultPath = "keystore/jwt-secret";
```

Comment:
Also, let's rename `defaultPath` to `oldPath` as we already have `newPath` there.